src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -64,6 +64,7 @@
                 </template>
                 <v-list dense>
                   <v-list-item-group color="primary">
+                    <!-- NB: this menu item is disabled in current release -->
                     <v-list-item disabled>
```

Comment:
Was this partly why you  were inquiring about Launch darkly??

Reply:
Yes, this would have been a good candidate for LD. Instead I just disabled it for the last release.

Reply:
Definitely, i would like to work with LD a bit as well!

Reply:
@rarmitag or @JohnamLane can setup the LD Flag for you. Could you work with Richard to do this?

Reply:
Will discuss with them, thanks!